applications/nrf5340_audio/README.rst

```diff
@@ -1227,6 +1230,34 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 
 .. _nrf53_audio_app_porting_guide:
 
+Testing FOTA upgrades
+=====================
+
+`nRF Connect Device Manager`_ can be used for testing FOTA upgrades.
+Complete the following steps to test upgrading the firmware on both cores at the same time on a gateway device (the procedure is identical for upgrading headset firmware):
+
+1. Install `nRF Connect Device Manager`_ on your Android or iOS device.
+#. Connect an external flash shield to the gateway. See :ref:`nrf53_audio_app_configuration_configure_fota` for details.
+#. Make sure the gateway runs a firmware that supports DFU using external flash memory.
+#. Use the buildprog.py script to create a .zip folder containing new firmware for both cores.
+   This can be done by running the following command:
+
+   .. code-block:: console
+
+      python buildprog.py -c both -b debug -d gateway --pristine -m external
+
+   .. note::
+      If you use Linux, the ``python`` command must excute Python 3.
+
+#. Transfer the generated .zip file to your Android or iOS device. The file name should start with :file:`dev_gateway_build_debug_dfu_application`.
+   It is possible to use cloud services like Google Drive, Dropbox or iCloud.
+#. Open `nRF Connect Device Manager`_ and look for ``NRF5340_AUDIO_GW_DFU`` in the scanned devices window.
+#. Tap on ``NRF5340_AUDIO_GW_DFU`` and then on the downward arrow icon at the bottom of the screen.
+#. Under the ``Firmware Upgrade`` section tap ``SELECT FILE``.
+#. Select the .zip file you transferred to the device.
```

Comment:
```suggestion
#. Select the zip file you transferred to the device.
```

Reply:
Done. Maybe zip should be written with capital letters? Like zip = ZIP?